### PR TITLE
[4.1.x] update Cedar dependencies to 4.1.x

### PR DESCRIPTION
**Pull request by cdisselkoen**

### CODE REVIEW

File: cedar-rust-hello-world/Cargo.toml [hunk 1]
```diff
@@ -9,8 +9,7 @@ publish = false
 serde_json = "1.0"
 
 [dependencies.cedar-policy]
-version = "4.0.0"
+version = "4.1.0"
 git = "https://github.com/cedar-policy/cedar"
 branch = "main"
```
Comment (shaobo-he-aws) [inline cedar-rust-hello-world/Cargo.toml:14]: Maybe `release/4.1.x`?

**Review by shaobo-he-aws — APPROVED** [review]

File: cedar-rust-hello-world/Cargo.toml (hunk 1 above)
Comment (cdisselkoen) [reply cedar-rust-hello-world/Cargo.toml:14]: I was spooked by the line 15 warning not to change this haha

Comment (khieta) [reply cedar-rust-hello-world/Cargo.toml:14]: +1 to update branch

Comment (cdisselkoen) [reply cedar-rust-hello-world/Cargo.toml:14]: Updated

**Review by khieta — APPROVED** [review]

**Pull request merged**